quantum/template/base/keymaps/default/keymap.c

```diff
@@ -15,17 +15,28 @@
  */
 #include QMK_KEYBOARD_H
 
+// Defines names for use in layer keycodes and the keymap
+enum layer_names {
+    _BASE,
+    _FN
+}
```

Comment:
I don't know if it's just me, but my keymaps never compile without a semicolon here.

```suggestion
};
```

Reply:
🤦‍♂